### PR TITLE
Parent the attestation sdk correctly.

### PR DESCRIPTION
**Pull request by pallavit**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -603,6 +607,7 @@
                     -snippetpath ${project.basedir}/sdk/appconfiguration/azure-data-appconfiguration/src/samples/java
                     -snippetpath ${project.basedir}/sdk/confidentialledger/azure-security-confidentialledger/src/samples/java
                     -snippetpath ${project.basedir}/sdk/containerregistry/azure-containers-containerregistry/src/samples/java
+		    -snippetpath ${project.basedir}/sdk/attestation/azure-security-attestation/src/samples/java
```
Comment (srnagar) [inline pom.xml:610]: fix formatting

**Review by srnagar — APPROVED** [review]

**Review by LarryOsterman — APPROVED** [review]
LGTM, once the formatting issue is resolved.

**Pull request merged**